src/app.js

```diff
@@ -327,6 +331,25 @@ var run = function() {
 		})
 	});
 
+	$('.inputFile').on('change', function() {
+		var fileList = $('input.inputFile')[0].files;
+		for (var i = 0; i < fileList.length; i++) {
+			(function(file) {
+				var fileReader = new FileReader();
+				fileReader.onload = function(e) {
+					SOL_CACHE_FILE = fileKey(file.name);
+					window.localStorage[SOL_CACHE_FILE] = e.target.result;
+					updateFiles();
+				}
+				fileReader.readAsText(file);
+			})(fileList[i]);
+		}
+
+		$filesEl.animate({left: Math.max( (0 - activeFilePos() + (FILE_SCROLL_DELTA/2)), 0)+ "px"}, "slow", function(){
```

Comment:
This (and all the above `function()`s) don't conform to the new lint. Please run `semistandard app.js` for a list of errors (we haven't addressed the indentation errors it spits out, yet)